operator/config/network-policies/deploy.yaml

```
@@ -0,0 +1,493 @@
+  name: forklift-controller-policy
+  namespace: openshift-mtv
+spec:
+  egress:
+  - to:
+    - podSelector: {}
+  - to:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: openshift-dns
+  - ports:
+    - port: 6443
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)